### PR TITLE
Added Follow event handler.

### PR DESCRIPTION
**Pull request by Ben67366**
Sample for Follow Evnet.
#1

### CODE REVIEW

**Comment by dlemstra** [comment]
Thanks for the PR and helping me out 👍 Will take a look at it this weekend.

File: src/LineBotSample/EventHandlers/FollowEventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+using System;
+using System.Threading.Tasks;
+using Line;
+using LineBotSample.Configuration;
+
+namespace LineBotSample.EventHandlers
+{
+    public class FollowEventHandler : ILineEventHandler
+    {
+        private readonly LineBotSampleConfiguration configuration;
```
Comment (dlemstra) [inline src/LineBotSample/EventHandlers/FollowEventHandler.cs]: This is not used so we can remove it here and from the constructor.

File: src/LineBotSample/EventHandlers/FollowEventHandler.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+            try
+            {
+                var user = await lineBot.GetProfile(evt.Source.User);
+                userName = $"{user.DisplayName} ({user.UserId})";
+            }
+            catch (LineBotException)
+            {
+            }
+            catch (Exception)
```
Comment (dlemstra) [inline src/LineBotSample/EventHandlers/FollowEventHandler.cs]: We don't need this `catch`.

**Review by dlemstra — COMMENTED** [review]
Thanks for the pull request. Can you fix my two remarks so we can merge this?

**Comment by Ben67366** [comment]
Thanks for your comment, I've fixed that two remarks. :)

**Pull request merged**